### DISCUSSION TITLE
Enable Logins for a Password Pusher Deployment

### DISCUSSION THREAD

**Discussion by jollero** [pglombardo/PasswordPusher#276] [Q&A] [answered]
Hi,

Is there a way to add a login to the password push deployment? I've noticed that the original site (https://pwpush.com/) has a login and sign-up but the self-hosted deploy sites (i.e. heroku) do not have it. I'd really like to have that feature when we deploy our own. Is there a way to enable it or add it?

Thanks a lot in advance! 

Regards,
Joseph

**Comment by pglombardo** [comment]
Hi @jollero - It is possible but I haven't gotten to documenting it yet (and figuring out how to best support it).

I put in a summary on the steps [here](https://github.com/pglombardo/PasswordPusher/issues/265#issuecomment-964432942).  It involves having an SMTP server available and setting some environment variables.

Logins aren't supported in the ephemeral containers for obvious reasons so make sure that you are using a database backed instance first.

Let me know if you have any questions or issues!

**Comment by pglombardo** [comment, marked as answer]
The documentation to enable logins is now here: https://docs.pwpush.com/docs/self-hosted-configuration/#logins
